src/serializable/pvm/increaseBalanceTx.ts

```diff
@@ -0,0 +1,44 @@
+import { pack, unpack } from '../../utils/struct';
+import { BaseTx } from '../avax/baseTx';
+import type { Codec } from '../codec';
+import { serializable } from '../common/types';
+import { TypeSymbols } from '../constants';
+import { Id } from '../fxs/common';
+import { BigIntPr } from '../primitives';
+import { PVMTx } from './abstractTx';
+
+@serializable()
+export class IncreaseBalanceTx extends PVMTx {
```

Comment:
should we follow the same name as in `avalanchego` so we know they are the same thing?